### ISSUE TITLE
Nested options for bounding box in SymbolNode

### ISSUE THREAD

**Issue by Luisav1** [phetsims/build-a-nucleus#86]
Currently the options for SymbolNode are a bit messy since they include the options for BoundingBox too. It'd be nice to separate the BoundingBox options in a nested options block.
```
{
      chargeProperty: null,
      fill: 'white',
      symbolTextFill: 'black',
      protonCountDisplayFill: PhetColorScheme.RED_COLORBLIND,
      massNumberDisplayFill: 'black',
      boundingBoxStroke: 'black',
      tandem: Tandem.REQUIRED // TODO: How to support phet-brand and sub-instrumented components? This applies to all the commented out tandems
    }
```

**Comment by zepumph** [comment]
This is blocked by https://github.com/phetsims/shred/issues/32. Updating assignment and labels.

**Comment by zepumph** [comment]
Over in https://github.com/phetsims/build-a-nucleus/issues/80 we went a different direction instead of trying to use a SymbolNode for a new view case, so this is no longer needed. Closing
